src/App.tsx

```diff
@@ -5,7 +5,22 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [query, setQuery] = useState('');
+
+  const onSearch = (value: string) => setQuery(value);
+  const onAddMovie = (newMovie: Movie) => {
```

Comment:
```suggestion
  const handleSearch = (value: string) => setQuery(value);
  const handleAddMovie = (newMovie: Movie) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364